api/tests/unit/config/fixtures/data_fixtures/AppFixturesTest.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\unit\config\fixtures\data_fixtures;
+
+use App\DataFixtures\AppFixtures;
+use App\Entity\Question;
+use App\Entity\Quiz;
+use PHPUnit\Framework\TestCase;
+
+final class AppFixturesTest extends TestCase
+{
+
+    private $appFixtures;
+    private $objectManager;
+
+    protected function setUp(): void
+    {
+        $objectManager = $this->getMockBuilder('Doctrine\Persistence\ObjectManager')
+            ->onlyMethods(['flush', 'persist'])
+            ->getMockForAbstractClass();
+        $this->objectManager = $objectManager;
+        
+        $appFixtures = new AppFixtures;
+        $appFixtures = $this->invokeProperty($appFixtures, 'objectManager', $objectManager);
+        $this->appFixtures = $appFixtures;
+    }
+
+    public function invokeProperty(object &$object, string $propertyName, mixed $parameter): object
+    {
+        $reflection = new \ReflectionClass(get_class($object));
+        $property = $reflection->getProperty($propertyName);
+        $property->setAccessible(true);
+        $property->setValue($object, $parameter);
+
+        return $object;
+    }
+
+    public function testLoad()
+    {
+        $loadFunction = $this->appFixtures->load($this->objectManager);
+
+        self::assertNull($loadFunction);
+    }
+
+    public function testCreateQuizzes()
+    {
+        $createQuizzesFunction = $this->appFixtures->createQuizzes();
+
+        self::assertNull($createQuizzesFunction);
+    }
+
+    public function testCreateQuestions()
+    {
+        $html = require dirname(__DIR__) . '/../../../../config/fixtures/quizzes/html-quiz/quiz.php';
+
+        $quiz = new Quiz;
+        $quiz->setTitle('Test title');
+        $quiz->setSlug('test-slug');
+
+        $createQuestionsFunction = $this->appFixtures->createQuestions($html['questions'], $quiz);
+
+        self::assertNull($createQuestionsFunction);
+    }
+
+    public function testCreateAnswers()
+    {
+        $question1 = require dirname(__DIR__) . '/../../../../config/fixtures/quizzes/html-quiz/questions/question_1.php';
+        $question = new Question;
+        $question->setContent('Test content');
+
+        $createAnswersFunction = $this->appFixtures->createAnswers($question1[0]['answers'], $question);
+
+        self::assertNull($createAnswersFunction);
+    }
+
+    public function testGetDataSets()
+    {
+        $result = $this->appFixtures->getDataSets();
```

Comment:
Can we check the paths within the `$result` We need to make sure they are not  not empty.